rllib/algorithms/impala/impala.py

```diff
@@ -408,33 +409,6 @@ def gather_experiences_directly(workers, config):
     return train_batches
 
 
-# Update worker weights as they finish generating experiences.
```

Comment:
Not needed anymore (exec. plan API). Forgot to remove this in yesterday's PR.

Reply:
Can we open a separate pr for this.

Reply:
Actually, let's do it here. Removed this code in this PR, because it was still using the LearnerThread's `self.weights_updated` property, which I get rid of in this PR.